tests/unit/test_vowpal_wabbit.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import pytest
+from unittest import mock
+
+
+import pandas as pd
+
+from reco_utils.recommender.vowpal_wabbit.vw import VW
+
+
+@pytest.fixture(scope="module")
+def df():
+    return pd.DataFrame(
+        dict(user=[1, 3, 2], item=[8, 7, 7], rating=[1, 5, 3], timestamp=[1, 2, 3])
+    )
+
+
+@pytest.fixture(scope="function")
+def model():
+    model = VW(col_user="user", col_item="item", col_prediction="prediction", q="ui")
+    yield model
+    del model
+
+
+def test_vw_init_del():
+    model = VW()
```

Comment:
This line can be saved by passing the fixture function?

Reply:
here I want to directly control creation and destruction instead of relying on the fixture.